src/test/resources/org/mule/tools/devkit/sonar/checks/pom/ScopeProvidedInMuleDependenciesCheckTest-ScopeCompile-pom.xml

```diff
@@ -6,7 +6,7 @@
     <groupId>org.mule.tools.devkit.sonar</groupId>
     <artifactId>certification-plugin</artifactId>
     <packaging>sonar-plugin</packaging>
-    <version>SOME-SNAPSHOT</version>
```

Comment:
Hum... we should touch this file, this is from another test... not sure we're even using it in current test.